ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -15,23 +15,17 @@
 
 package io.confluent.ksql.tools.migrations;
 
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.confluent.ksql.api.client.Client;
+import io.confluent.ksql.api.client.ServerInfo;
 import io.confluent.ksql.properties.PropertiesUtil;
-import io.confluent.ksql.rest.client.KsqlRestClient;
```

Comment:
Are we able to remove the ksqldb-rest-client dependency in the pom for this module now that this is gone?